bundler.d/test.rb

```diff
@@ -6,6 +6,7 @@
   gem 'ci_reporter', '>= 1.6.3', "< 2.0.0", :require => false
   gem 'rdoc'
   gem 'minitest', '~> 4.7', :platforms => :ruby_19
```

Comment:
Should we also replace minitest with test-unit on Ruby 1.9 for consistency?

As I understand it (http://projects.theforeman.org/issues/2261), we only have this in here for Ruby 1.9.2.  It might be that we can use test-unit on 1.9.